src/components/LoadingSpinner.jsx

```
@@ -61,7 +38,8 @@ export default function LoadingSpinner() {
       <pre
         className="LoadingCodeBlock"
         dangerouslySetInnerHTML={{
```

Biome (2.1.2)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)